pyaedt/hfss.py

```diff
@@ -3036,6 +3046,8 @@ def create_boundary(
             Name of the boundary. The default is ``""``.
         is_infinite_gnd : bool, optional
             Whether the boundary is an infinite ground. The default is ``False``.
+        props : dict, optional
+            Additional properties for the boundary. The default is an empty dict ``{}``.
```

Comment:
```suggestion
            Additional properties for the boundary. The default is ``{}``.
```
Given that the data type is "dict," we assume that a developer knows that the default of ``{}`` is an empty dictionary.